drivers/display/display_renesas_lcdc.c

```diff
@@ -0,0 +1,595 @@
+	 * and the system enters the low-power state (i.e. ARM WFI) when possible.
+	 *
+	 * XXX CONFIG_PM_DEVICE_RUNTIME is no supported yet!
+	 */
+	da1469x_pd_acquire_noconf(MCU_PD_DOMAIN_SYS);
+#endif
+
+	return 0;
+}
+
+static int display_smartbond_blanking_on(const struct device *dev)
+{
+	const struct display_smartbond_config *config = dev->config;
+	struct display_smartbond_data *data = dev->data;
+	int ret = 0;
+
+	k_sem_take(&data->device_sem, K_FOREVER);
+
+	/*
+	 * This bit will force LCD controller's output to blank that is,
+	 * the controller will keep operating without outputting any
+	 * pixel data.
+	 */
+	LCDC->LCDC_MODE_REG |= LCDC_LCDC_MODE_REG_LCDC_FORCE_BLANK_Msk;
+
+	/* If enabled, disable display port. */
+	if (gpio_is_ready_dt(&config->disp)) {
+		ret = gpio_pin_configure_dt(&config->disp, GPIO_OUTPUT_INACTIVE);
+		if (ret < 0) {
+			LOG_WRN("Display port could not be de-activated");
+		}
+	}
+
+	k_sem_give(&data->device_sem);
+
+	return ret;
+}
+
+static int display_smartbond_blanking_off(const struct device *dev)
+{
+	const struct display_smartbond_config *config = dev->config;
+	struct display_smartbond_data *data = dev->data;
+	int ret = 0;
+
+	k_sem_take(&data->device_sem, K_FOREVER);
+
+	/* If used, enable display port */
+	if (gpio_is_ready_dt(&config->disp)) {
+		ret = gpio_pin_configure_dt(&config->disp, GPIO_OUTPUT_ACTIVE);
+		if (ret < 0) {
+			LOG_WRN("Display port could not be activated");
+		}
+	}
+
+	/*
+	 * This bit will force LCD controller's output to blank that is,
+	 * the controller will keep operating without outputting any
+	 * pixel data.
+	 */
+	LCDC->LCDC_MODE_REG &= ~LCDC_LCDC_MODE_REG_LCDC_FORCE_BLANK_Msk;
+
+	k_sem_give(&data->device_sem);
+
+	return ret;
+}
+
+static int display_smartbond_set_orientation(const struct device *dev,
+			const enum display_orientation orientation)
+{
+	/* Orientation is not available in display panels. */
+	if (orientation == DISPLAY_ORIENTATION_NORMAL) {
+		return 0;
+	}
+
+	return -ENOTSUP;
+}
+
+static void *display_smartbond_get_framebuffer(const struct device *dev)
+{
+	struct display_smartbond_data *data = dev->data;
+
+	return ((void *)data->buffer);
+}
+
+static void display_smartbond_get_capabilities(const struct device *dev,
+			struct display_capabilities *capabilities)
+{
+	const struct display_smartbond_config *config = dev->config;
+
+	memset(capabilities, 0, sizeof(*capabilities));
+
+	/*
+	 * Multiple color formats should be supported by LCDC. Currently, RGB56 and ARGB888
+	 * exposed by display API are supported. In the future we should consider supporting
+	 * more color formats which should require changes in LVGL porting.
+	 * Here, only one color format should be supported as the frame buffer is accessed
+	 * directly by LCDC and is allocated statically during device initialization. The color
+	 * format is defined based on the pixel-format property dictated by lcd-controller
+	 * bindings.
+	 */
```

Comment:
Please be consistent here as far as multi-instance versus single instance code. For example, if you wanted to make this block entirely multi instance, it would look something like this:

```
capabilities->supported_pixel_formats = config->pixel_format
capabilities->current_orientation = DISPLAY_ORIENTATION_NORMAL;
capabilities->current_pixel_format = config->pixel_format
capabilities->x_resolution = config->x_res;
capabilities->y_resolution = config->y_res;
```

Alternatively, an entirely single instance version would look like this:

```
capabilities->supported_pixel_formats = DT_INST_PROP(0, pixel_format);
capabilities->current_orientation = DISPLAY_ORIENTATION_NORMAL;
capabilities->current_pixel_format = DT_INST_PROP(0, pixel_format);
capabilities->x_resolution = DT_INST_PROP(inst, width);
capabilities->y_resolution = DT_INST_PROP(inst, height);
```

Personally, I would *strongly* prefer the driver is implemented as multi instance, since that is more scalable in the future.